src/lib/parse/compoundValue.cpp

```diff
@@ -162,6 +157,8 @@ void compoundValueEnd(ConnectionInfo* ciP, ParseData* parseDataP)
   // latest contextAttribute, i.e. the attribute whose 'contextValue' is the
   // owner of this compound value tree.
   //
+  // FIXME P4 #1862: obsolete reference to XML in the above comment?
```

Comment:
I looked into the whole lastContextAttribute mechanism.
It is very much needed, in the JSON v1 parse.
JSON v2 parse is completely different and no lastContextAttribute pointer is needed.

So, just change **XML** for **JSON V1** in the comment.